modules/core/include/opencv2/core/hal/intrin_vsx.hpp

```diff
@@ -1363,8 +1364,19 @@ OPENCV_HAL_IMPL_VSX_TRANSPOSE4x4(v_uint32x4, vec_uint4)
 OPENCV_HAL_IMPL_VSX_TRANSPOSE4x4(v_int32x4, vec_int4)
 OPENCV_HAL_IMPL_VSX_TRANSPOSE4x4(v_float32x4, vec_float4)
 
-CV_CPU_OPTIMIZATION_HAL_NAMESPACE_END
+/* Permute 32B into a 16B vector using a control vector. Any control byte value greater than the number of
```

Comment:
There should be plain scalar implementation for all intrinsics in `intrin_cpp.hpp` file. Intrinsic documentation should be added to that file as well.

Reply:
I think I may be overlooking it, but how would I build to include the simulated intrinsics?

Reply:
Simulated intrinsics file is used to produce doxygen documentation(`BUILD_DOCS` cmake flag that leads to definition of `CV_DOXYGEN` to 1 and exclusion of non-simulated intrinsics). Another option to use simulated intrinsics is `CPU_BASELINE` cmake flag. It's possible to set it to empty value to prohibit usage of any platform specific features that disable non-simulated intrinsics for any platform as well.